structuralcodes/sections/_rectangularRC.py

```diff
@@ -0,0 +1,580 @@
+        if self.diameter_top is None:
+            object.__setattr__(self, 'diameter_top', self.diameter_corner)
+        if self.diameter_left is None:
+            object.__setattr__(self, 'diameter_left', self.diameter_corner)
+        if self.diameter_right is None:
+            object.__setattr__(self, 'diameter_right', self.diameter_corner)
+
+    def get_as_tot(self) -> float:
+        as_tot: float = [self.diameter_bottom, self.diameter_corner, self.diameter_left, self.diameter_right, self.diameter_top] * [self.number_bottom, 4, self.number_left, self.number_right, self.number_top]
+
+
+class RectangularRC(Section):
+    """Class for RC rectangular cross section.
+    The class assumes width and height respectively
+    along local x and y axes.
+
+    Input parameters:
+    width: width of the section
+    height: height of the section
+    cover: net cover (distance from outer edget of the section to
+    edge of the stirrup)
+    reinforcement_data: ReinforcementData
+    concrete: Concrete
+    reinforcement: Reinforcement (TODO!!!!!! for now const law)
+    confined_concrete: Concrete (default = None) When set to None the
+    same concrete is adopted for concrete core. When using the string
+    "auto" the confinement will be automatically computed
+    stirrups_data: StirrupsData (default = None) to use when need
+    automatic computation of concrete confinement
+    name: optional string for identidying the section
+
+    Axes definition:
+
+                     z ▲
+                       │
+                       │
+            ┌──────────┼───────────┐
+            │          │           │
+            │          │           │
+            │          │           │
+            │          │           │
+            │          │           │
+            │          │           │
+            │          │           │
+            │          └───────────┼─────►
+            │                      │     y
+            │                      │
+            │                      │
+            │                      │
+            │                      │
+            │                      │
+            │                      │
+            └──────────────────────┘
+
+            RectangularRCSection.fromGeometry(Geometry(itPolygons,itMaterials))
+
+
+    """
+
+    def __init__(
+        self,
+        width: float,
+        height: float,
+        cover: float,
+        reinforcement_data: ReinforcementData,
+        concrete: Concrete,
+        reinforcement: ElasticPlastic,
+        confined_concrete: t.Union[Concrete, str, None] = None,
+        stirrups_data: t.Optional[StirrupsData] = None,
+        name: t.Optional[str] = None,
+    ) -> None:
+        if name is None:
+            name = 'RectangularRCSection'
+        super().__init__(name)
+
+        # section geometry
+        self.height = height
+        self.width = width
+        self.cover = cover
+        self.geometry: SurfaceGeometry
```

Comment:
Here is where I start to change the class you started on. From what I see, there is not much to change or add